### ISSUE TITLE
[BUG] Problem in using multiprocessing with model

### ISSUE THREAD

**Issue by erezinman** [salesforce/Merlion#42]
In version 1.0.1, I try to pass models (say `DefaultForecaster(DefaultForecasterConfig('10min'))`, but it happens other mnodels as well) to different process using the native python multiprocessing package (v3.6.9), and I get the following error:

> 'DefaultForecaster' object has no attribute 'config'. 'config' is an invalid kwarg for the load()

**Comment by aadyotb** [comment]
Can you please provide a full code example and full error message?

**Comment by erezinman** [comment]
@aadyotb Yes.

```python
from merlion.models.defaults import *
import multiprocessing as mp

with mp.Pool(processes=1) as pool:
    for _ in pool.imap_unordered(print, [DefaultForecaster(DefaultForecasterConfig('10min'))]):
        pass
```
> Process ForkPoolWorker-30:
> Traceback (most recent call last):
>   File "/usr/lib/python3.6/multiprocessing/process.py", line 258, in _bootstrap
>     self.run()
>   File "/usr/lib/python3.6/multiprocessing/process.py", line 93, in run
>     self._target(*self._args, **self._kwargs)
>   File "/usr/lib/python3.6/multiprocessing/pool.py", line 108, in worker
>     task = get()
>   File "/usr/lib/python3.6/multiprocessing/queues.py", line 337, in get
>     return _ForkingPickler.loads(res)
>   File "/usr/local/lib/python3.6/dist-packages/merlion/models/base.py", line 185, in \_\_setstate\_\_
>     f"'{type(self).\_\_name\_\_}' object has no attribute '{name}'. "
> AttributeError: 'DefaultForecaster' object has no attribute 'config'. 'config' is an invalid kwarg for the load() method.

**Comment by aadyotb** [comment]
@erezinman thanks for surfacing this bug. The problem should be fixed with Merlion v1.1.0 (#47).
